themes/bootstrap3/templates/feedback/form.phtml

```diff
@@ -73,7 +73,7 @@
       <?php if (in_array($el['type'], ['checkbox', 'radio'])): ?>
         <div class="field-set" role="group"<?= !empty($el['label']) ? ' aria-labelledby="' . $this->escapeHtmlAttr($el['name']) . '"' : ''?>>
       <?php else: ?>
-        <div class="field-set">
+        <div class="field-set" role="contentinfo" aria-label="<?=$elementHelpPre ?? $elementHelpPre?>">
```

Comment:
Tuossa ehdossa ei ole järkeä, kun `??` molemmilla puolilla on sama asia. $elementHelpPre:tä ei voi käyttää labelina, koska siinä voi olla html:ää.

Reply:
Eh joo, kuumuus vissiin sekoitti pään ja poistin oikealta puolelta tyhjän vaihtaessani ??-ehtoon.
Laitan nyt alkuun draftiksi ja katson saanko upstreamiin aikaiseksi PR:n
`<?=$this->escapeHtmlAttr($elementHelpPre) ?? ''?>` -tyylisesti ja poistan sitten tämän.